README.md

```diff
@@ -507,6 +507,8 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
       - mini.5p--3p.bam: subsample of hifi reads from the public pacbio dataset [pbmc singlecell mini](https://downloads.pacbcloud.com/public/dataset/IsoSeq_sandbox/2022_pbmc_singlecell_mini/ccs.bam) where the primers where removed with LIMA.
       - test.sorted.bam: 35 reads from a [public PacBio Revio dataset of HG002](https://downloads.pacbcloud.com/public/revio/2022Q4/HG002-rep1/analysis/HG002.m84011_220902_175841_s1.GRCh38.bam) aligned to GRCh38.
       - test.sorted.bam.bai: BAM index for 'test.sorted.bam'
+      - sample_pacbio_longread_dna.aligned.bam: WGS aligned reads generated by pbmm2, containing RG information
```

Comment:
Whas is the source of the reads used to align, and what is the reference genome - is it the `genome.fasta` here: https://github.com/nf-core/test-datasets/tree/1991fd4939c86871d116eb74276b2f00b5388d65/data/genomics/homo_sapiens/genome

For example?

Reply:
You could also consider re-aligning `test.sorted.bam` to genome.fasta with pbmm2 and updating it (with RGs), if that file contains enough reads for you. The missing RGs in `test.sorted.bam` was an oversight by me.

Reply:
Hi - I changed the datasets so they were based off the publicly available https://www.pacb.com/connect/datasets/ PURETARGET REPEAT EXPANSION PANEL WITH CORIELL SAMPLES. 
The data is aligned to genome3.fasta which is the segment of hg38.fa that corresponds to (chr19:45760000-45770300. In addition to the test.sorted.bam not containing read groups, the 35 were and aligned to hg38.fa, and corresponded to many different chromosomes/locations, + I found adding RG info to this dataset was not super trivial, so I opted to create new sample data for the purpose of testing pbsv.